intl/en/messages.json

```diff
@@ -0,0 +1,3 @@
+{
+  "5bdcd0e36db5b1100819f9aa25e9bdf0": "Cloudant {{regex}} syntax does not support global"
+}
```

Comment:
its good now, reruning global slt-globalize -e did it now. probably because of the bug this morning during the demo with ./node_modules...